### ISSUE TITLE
Imposible to select multiple entries in orphan resource explorer at once

### ISSUE THREAD

**Issue by qarmin** [godotengine/godot#30211]
**Issue description:**
I can't select more than one entry in orphan resource explorer.
It is very frustrating, when I working on big project.

![Zrzut ekranu z 2019-06-30 22-01-54](https://user-images.githubusercontent.com/41945903/60401550-87084d00-9b83-11e9-9bff-287700d45efd.png)

**Comment by jambalayadog** [comment]
Yes, I have just run into this issue.  It would be great to select the entire contents of a folder by selecting the folder.

Or when deleting a folder from the File Explorer, and it prompts you that files are in use, an option to exclude in-use files would be great.

**Comment by jamonholmgren** [comment]
This is fixed, issue can be closed.

<img width="800" alt="Orphan Resource Explorer" src="https://github.com/user-attachments/assets/63f6240b-16e8-4339-a446-c5a49c0df5e2" />
